docs/product-installation-guide/src/main/asciidoc/clustering-artemis-activate-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='clustering-artemis-activate-proc']
+= Activating ActiveMQ JMS broker
+
+.Prerequisite
+Elasticsearch installed on the main node of the cluster
+
+After you install Elasticsearch, you must activate ActiveMQ JMS broker on the main node of the cluster.
+
+.Procedure
+
+. Add the following properties to the `_EAP_HOME_/bin/./standalone.sh` file:
```

Comment:
please change the file path to  EAP_HOME/standalone/configuration/standalone.xml